async-nats/CHANGELOG.md

```diff
@@ -1,3 +1,64 @@
+# 0.32.0
+## Overview
+
+This PR focuses on two things:
```

Comment:
Did you mean to say, "this release primarily focuses on two things"

Reply:
yes. fixed.